src/tests/shambackends/kernel_call_tests.cpp

```
@@ -65,7 +77,53 @@ NEW_TEST(Unittest, "shambackends/kernel_call", 1) {
+    P_field.fill(0);
+    cs_field.fill(0);
+
+    sham::kernel_call(
+        dev_sched->get_queue(),
+        sham::MultiRefOpt{rho_field, uint_field, sham::empty_buf_ref<T>()},
+        sham::MultiRefOpt{P_field, cs_field},
+        size,
+        [](u32 i,
+           const T *__restrict rho,
+           const T *__restrict U,
+           const T *__restrict should_be_nullptr /*unused*/,
```

Cppcheck (2.21.0)

[style] The function 'assert_float_equal' is never used.

(unusedFunction)